yaroslava_skomarovska/src/HelloWorld.java

```diff
@@ -0,0 +1,7 @@
+public class HelloWorld {
+public static void main(String[] args) {
+    System.out.println("Hello World");
+
```

Comment:
Please don't leave empty lines between code and closing braces. I haven't mention this before but it's not java codestyle